hbase-server/src/main/java/org/apache/hadoop/hbase/util/JvmVersion.java

```diff
@@ -44,4 +44,13 @@ public static boolean isBadJvmVersion() {
     String version = System.getProperty("java.version");
     return version != null && BAD_JVM_VERSIONS.contains(version);
   }
+
+  /**
+   * Return the current JVM version information.
+   */
+  public static String getVersion() {
```

Comment:
According to the javadoc in the class header, this is a repurposing of the existing class. I think it's fine to put this method into this class, as the name and location of the class seem like a reasonable place to find this information. Can you update the javadoc on the class header according to the expanded scope of the class?

Otherwise, +1.